src/SignalR/clients/csharp/Http.Connections.Client/src/WebSocketConnectionContext.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Http.Connections.Client
+{
+    /// <summary>
+    /// Used to make a connection to an SignalR using a WebSocket-based transport.
+    /// </summary>
+    public sealed class WebSocketConnectionContext
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WebSocketConnectionContext"/> class.
+        /// </summary>
+        /// <param name="uri">The URL to connect to.</param>
+        /// <param name="options">The connection options to use.</param>
+        public WebSocketConnectionContext(Uri uri, HttpConnectionOptions options)
```

Comment:
i want to confirm here. so init properties can be initialized only on construction phase but it doesn't mean that they must be initialized. if i remove that constructor, then compiler wants Url & Options properties to be nullable since nullalbes are enabled. And we don't have constuctor that forces values in these properties.

if that is intention, i can remove constructor, and make both properties as nullable

Reply:
Ah, this is my first time using init, thought they also meant required, but they don't. Let's remove the init (don't add set), and keep the ctor.